### PR TITLE
Add Internationalization Considerations section

### PR DESCRIPTION
**Pull request by mprorock**
***
<a href="https://pr-preview.s3.amazonaws.com/mesur-io/vc-jwt/pull/92.html" title="Last updated on Jun 6, 2023, 9:38 PM UTC (ea6d3f8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/92/e3ecd2e...mesur-io:ea6d3f8.html" title="Last updated on Jun 6, 2023, 9:38 PM UTC (ea6d3f8)">Diff</a>

### CODE REVIEW

**Review by andresuribe87 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -395,6 +395,23 @@ <h2>Security Considerations</h2>
...
+    internationalization considerations when publishing data described
+    in this specification. Disregarding internationalization hampers the
+    production and consumption of data across diverse languages and
+    societies, thereby restricting the applicability and significantly
+    reducing the value of the specification as a standard.
+  </p>
+  <p>
+    Implementers are advised to note and abide by all
+    internationalization considerations called out in the
+    [[VC-DATA-MODEL]].
```
Comment (selfissued) [inline index.html:412]: Are there W3C Internationalization spec(s) that we should also directly reference?

**Review by selfissued — APPROVED** [review]
Approved with comments

File: index.html [hunk 2]
```diff
@@ -395,6 +395,23 @@ <h2>Security Considerations</h2>
   </p>
 </section>
 
+
+<section class="informative">
+  <h2>Internationalization Considerations</h2>
+  <p>
+    Technical implementers should take into account several
```
[the remark this answers is not in the document]
Comment (TallTed) [reply index.html:402]: One-click mergeable suggestion...
```suggestion
    Implementers should take into account several
```

File: index.html (hunk 1 above)
Comment (OR13) [inline index.html:412]:
```suggestion
    [[VC-DATA-MODEL]], see also <a href="https://www.w3.org/standards/webdesign/i18n">i18n</a>.
```

**Review by OR13 — APPROVED** [review]

**Pull request merged**